action.yml

```diff
@@ -38,7 +38,7 @@ runs:
       if: github.event_name != 'pull_request' || github.event.action != 'closed'
       run: |
         result=$(git diff --cached)
-        echo "::set-output name=result::$result"
+        echo "result=$result" >> $GITHUB_OUTPUT
```

Comment:
他のリポジトリで以下のように落ちているので、修正した方が良いかも？

https://github.com/dev-hato/deploy-webhook/actions/runs/3243580534/jobs/5318501548

```
.github/workflows/docker-compose-build.yml:86:9: shellcheck reported issue in this script: SC2086:info:4:36: Double quote to prevent globbing and word splitting [shellcheck]
   |
86 |         run: |
   |         ^~~~
```

Reply:
```suggestion
        echo "result=$result" >> "${GITHUB_OUTPUT}"
```